packages/experimental/src/components/AboutModal/AboutModal.test.js

```diff
@@ -140,36 +143,95 @@ describe('AboutModal', () => {
     const { click } = fireEvent;
     const tabToSelect = screen.getByRole('tab', { name: /Version number/i });
     click(tabToSelect);
-    expect(tabToSelect.classList.contains('bx--tabs__nav-item--selected'));
+    expect(
+      tabToSelect.parentElement.classList.contains(
+        'bx--tabs__nav-item--selected'
```

Comment:
Should we use the carbon prefix in tests?

Reply:
Probably. I'll do that under #393.